pkg/cmd/render/render.go

```diff
@@ -63,6 +81,18 @@ func (r *renderOpts) Validate() error {
 		return err
 	}
 
+	// Validate all files are specified when specifying infrastructure and configmap files
+	if len(r.clusterInfrastructureInputFile) != 0 || len(r.cloudProviderConfigOutputFile) != 0 {
```

Comment:
````
if infra, provider := len(r.clusterInfrastructureInputFile) != 0, len(r.cloudProviderConfigOutputFile); infra || provider {
  if infra && provider {
```

makes it easier to read

Reply:
This is nice. I've updated the syntax accordingly. Thanks for introducing me to this ;-)